src/context/apiClient/apiClientContextController/interceptors/responseInterceptors.ts

```diff
@@ -20,7 +19,7 @@ export const responseFailureInterceptor = async (error: AxiosError<unknown>) =>
     authStorage.expires = null;
     authStorage.refreshToken = null;
 
-    window.location.replace(AppRoute.login);
+    window.location.replace('/login');
```

Comment:
I think this is not really optimal, as there is no typesaftey. But if we implement https://headstart.atlassian.net/browse/ZN-571 we could move it somewhere into react lifecycle as we would be subscribed to authStorage values